tests/metrics/density/memory_usage.sh

```diff
@@ -194,32 +199,45 @@ function get_individual_memory(){
 	read -r -a second_values <<< "${second_process_result}"
 	read -r -a third_values <<< "${third_process_result}"
 
+	declare -a fv_array
+	declare -a sv_array
+	declare -a tv_array
+
+	# remove null values from arrays of results
+	for ((i=0;i<"${NUM_CONTAINERS}";++i)); do
```

Comment:
Just curious, why would you get null values for memory in the first place? Would be useful to add a comment regarding this.